db/migrate/20190725194654_add_grant_creator_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddGrantCreatorToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :grant_creator, :boolean, default: false
```

Comment:
This needs a not null constraint along with the default false.

Reply:
Updated.